app/controllers/organizations_controller.rb

```diff
@@ -19,6 +19,9 @@ def new
 
   def create
     @organization = Organization.new(new_organization_params)
+    if Classroom.flipper[:explicit_assignment_submission].enabled? current_user
+      @organization.setup_webhook(webhook_events_url)
```

Comment:
@cyhsutw We should probably make it a service object, I'd really rather not put the logic for this in a controller.